lib/runtime-core/src/export.rs

```diff
@@ -11,6 +11,8 @@ pub enum Context {
     Internal,
 }
 
+unsafe impl Send for Context {}
```

Comment:
Would it be possible to write a comment explaining why this is safe? Or why it can't be inferred?